microhapulator/pipeaux.py

```diff
@@ -153,16 +153,14 @@ def final_html_report(
             mhpl8rversion=microhapulator.__version__,
             samples=samples,
             summary=summary,
-            plots=plots,
             thresholds=thresholds,
-            zip=zip,
             read_length_table=read_length_table,
             typing_rates=typing_rates,
             mapping_rates=mapping_rates,
             markernames=marker_names,
             qc=qc,
-            len=len,
-            isna=pd.isna,
```

Comment:
Jinja apparently has a very limited Python runtime, so references to these functions were passed so that they could be called in the Jinja template. Since they are no longer called, we can safely remove these references.